central/complianceoperator/manager/manager.go

```diff
@@ -248,7 +248,7 @@ func (m *managerImpl) addProfileNoLock(profile *storage.ComplianceOperatorProfil
 		}
 
 		if err := m.registerCheckFromRule(standard.ID, profileProductType, fullRule); err != nil {
-			return errors.Wrapf(err, "registering check %s", fullRule.GetName())
+			return fmt.Errorf("registering check %s: %w", fullRule.GetName(), err)
```

Comment:
is there a push to stop using `github.com/pkg/errors`? Scanner team has been trying to make sure it's not used in `scanner/` nor anything in `pkg/` related to Scanner V4, but I figured the rest of the product would still exclusively use the former.

Reply:
Not that I know of, I think your assessment is correct that it's still widely used in our code. Here I decided to change it up to avoid a `pkgErrors.Wrap` situation. Usually I try to follow the convention of the module, but I would be in favor of getting rid of `github.com/pkg/errors`.